Automattic/wp-calypso
# Account closure: handle delete button click and logout on success

This PR connects up the delete button in Me > Account > Close.

The user is logged out if the account closure API call is successful.

Todo:
- [x] Reducer
- [x] Selector
- [x] Connect up action to button
- [x] Connect selector and trigger logout when isClosed = true 
- [x] Dispatch action for successful closure

### To test

Login as a user with no purchases and no Atomic sites (you may want to create a fresh test user with https://wordpress.com/start/account).

Head to:

http://calypso.localhost:3000/me/account/close

Click 'Delete Account' and fill in your username in the confirm dialog. Submit the form, and confirm that you're logged out, and are not able to re-login.
